README.md

```diff
@@ -81,6 +86,20 @@ $ driftctl scan --from tfstate://terraform.tfstate
 $ driftctl scan --from tfstate+s3://my-bucket/path/to/state.tfstate
 ```
 
+Or with ou docker image:
+```bash
+docker run \
+-v ~/.aws:/app/.aws:ro \
```

Comment:
Could you add two spaces indent for each newline, I think it will improve readability.

Reply:
:heavy_check_mark: